src/hotspot/os/linux/os_linux.cpp

```diff
@@ -3272,7 +3272,7 @@ struct bitmask* os::Linux::_numa_nodes_ptr;
 struct bitmask* os::Linux::_numa_interleave_bitmask;
 struct bitmask* os::Linux::_numa_membind_bitmask;
 
-bool os::pd_uncommit_memory(char* addr, size_t size) {
+bool os::pd_uncommit_memory(char* addr, size_t size, bool exec) {
```

Comment:
nit: I'm irritated by `bool exec` in `pd_uncommit_memory`, but `bool executable` in `pd_reserve_memory`. Choose one :-)

Reply:
Thanks, fixed. Too much of code shuffling. Now these should be consistent with the surroundings.